optuna/samplers/_tpe/sampler.py

```diff
@@ -457,6 +475,21 @@ def _get_internal_repr(
                     values[param_name].append(distribution.to_internal_repr(param))
         return {k: np.asarray(v) for k, v in values.items()}
 
+    def _extend_parzen_estimators_and_quantiles_for_ctpe(
```

Comment:
This method has side effect, i.e. to change some arguments inside the method rather than return them, and this is not really a good practice.